openapi/openapi_spec.json

```diff
@@ -11677,14 +11673,22 @@
                 "content-length",
                 "1024"
               ]
-            ]
+            ],
+            "nullable": true
           },
           "status_code": {
             "type": "integer",
             "format": "int32",
             "description": "The HTTP status code for the webhook sent.",
             "example": 200,
+            "nullable": true,
             "minimum": 0
+          },
+          "error_message": {
+            "type": "string",
+            "description": "Error message in case any error occurred when trying to deliver the webhook.",
+            "example": 200,
+            "nullable": true
           }
         }
       },
```

Comment:
openapi documentation is not generated for the new endpoint.

Reply:
+1

Reply:
Done, thanks for pointing this out!